releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,90 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 4.0.0-preview2 - March 7, 2024
+
+### Feature Service Injector
+
+Dependency injection can be wired up with a variant feature flag. Different implementations of a service interface can be considered as variant services. Using the newly provided `IVariantServiceProvider<TService>`, the appropriate service can be injected.
```

Comment:
I'd suggest opening with something like "It is now possible to use variant feature flags to control which implementation of a service dependency injection uses."

Reply:
Agreed. Stole a lot from https://github.com/microsoft/FeatureManagement-Dotnet/pull/388/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R977 , but this should be more about announcing it rather than using it.

Reply:
> this should be more about announcing it rather than using it.

Agree. I will update the README.

Reply:
Went with:

> It is now possible to use variant feature flags to control which implementation of a service is used in dependency injection. This enables different implementations to be used for different audiences of the application. After using the newly provided `WithVariantService<TService>` builder method, a `IVariantServiceProvider<TService>` can be injected which will supply the appropriate variation of `TService` determined by allocation.

And ripped out all code snippets about it.